test/apps/iodemo/io_demo.cc

```diff
@@ -1754,7 +1860,7 @@ static int parse_args(int argc, char **argv, options_t *test_opts)
         case 'q':
             test_opts->validate = true;
             break;
-        case 'a':
+        case 'A':
```

Comment:
update help msg, run_io_demo.sh, and azure scripts

Reply:
done